services/utils.py

```diff
@@ -55,15 +55,18 @@ def user_media_url(what):
 }
 
 
-version_re = re.compile(r"""(?P<major>\d+)         # major (x in x.y)
+version_re = re.compile(
+    r"""(?P<major>\d+)         # major (x in x.y)
                             \.(?P<minor1>\d+)      # minor1 (y in x.y)
                             \.?(?P<minor2>\d+|\*)? # minor2 (z in x.y.z)
                             \.?(?P<minor3>\d+|\*)? # minor3 (w in x.y.z.w)
                             (?P<alpha>[a|b]?)      # alpha/beta
                             (?P<alpha_ver>\d*)     # alpha/beta version
                             (?P<pre>pre)?          # pre release
                             (?P<pre_ver>\d)?       # pre release version
-                          """, re.VERBOSE)
+                          """,
```

Comment:
nit: the formatting for this is weird now (not that it matters after it's compiled)